.travis/install.sh

```diff
@@ -34,6 +34,10 @@ then
     then
         pyenv install 3.7.0
         pyenv global 3.7.0
+    elif [ "${TOXENV}" = "py39" ]
+    then
+        pyenv install 3.9
+        pyenv global 3.9
```

Comment:
this needs to match the travis.yml, which uses py38 (not py39).

Reply:
added another conditional branch here, but i don't knwo why it's not exposed in the canges ... even because here it's present https://github.com/borgbackup/borg/pull/5373/commits/a9ccdc9033aa81c519efc35be8b9e53daa64a466 🤷‍♂️